src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersMappingBuilderContext.cs

```diff
@@ -66,6 +76,37 @@ private HashSet<string> InitIgnoredUnmatchedProperties(IEnumerable<string> allPr
         return unmatched;
     }
 
+    private IEnumerable<string> GetIgnoredObsoleteTargetMembers()
+    {
+        var obsoleteStrategy = BuilderContext.Configuration.Properties.IgnoreObsoleteMembersStrategy;
+
+        if (!obsoleteStrategy.HasFlag(IgnoreObsoleteMembersStrategy.Target))
+            return Enumerable.Empty<string>();
+
+        var obsoleteAttribute = BuilderContext.Types.Get<ObsoleteAttribute>();
+        return Mapping.TargetType
+            .GetAccessibleMappableMembers()
+            .Where(x => x.MemberSymbol.HasAttribute(obsoleteAttribute))
+            .Select(x => x.Name);
+    }
+
+    private IEnumerable<string> GetIgnoredObsoleteSourceMembers()
+    {
+        var obsoleteStrategy = BuilderContext.Configuration.Properties.IgnoreObsoleteMembersStrategy;
+
+        if (obsoleteStrategy.HasFlag(IgnoreObsoleteMembersStrategy.Source))
```

Comment:
I'd invert this condition to improve the readability. Similar to the style which is implemented by `GetIgnoredObsoleteTargetMembers`.